bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/camera/CameraHandler.java

```diff
@@ -1,161 +1,161 @@
-        }
-        return super.getNAThingProperty(channelId);
-    }
-
-    protected State getStatusState() {
-        return module != null ? toOnOffType(module.getStatus()) : UnDefType.UNDEF;
-    }
-
-    protected State getSdStatusState() {
-        return module != null ? toOnOffType(module.getSdStatus()) : UnDefType.UNDEF;
-    }
-
-    protected State getAlimStatusState() {
-        return module != null ? toOnOffType(module.getAlimStatus()) : UnDefType.UNDEF;
-    }
-
-    protected State getIsLocalState() {
-        return module != null ? toOnOffType(module.getIsLocal()) : UnDefType.UNDEF;
-    }
-
-    protected State getLivePictureURLState() {
-        String livePictureURL = getLivePictureURL();
-        return livePictureURL == null ? UnDefType.UNDEF : toStringType(livePictureURL);
-    }
-
-    protected State getLivePictureState() {
-        String livePictureURL = getLivePictureURL();
-        return livePictureURL == null ? UnDefType.UNDEF : HttpUtil.downloadImage(livePictureURL);
-    }
-
-    protected State getLiveStreamState() {
-        String liveStreamURL = getLiveStreamURL();
-        return liveStreamURL == null ? UnDefType.UNDEF : new StringType(liveStreamURL);
-    }
-
-    /**
-     * Get the url for the live snapshot
-     *
-     * @return Url of the live snapshot
-     */
-    private String getLivePictureURL() {
-        String result = getVpnUrl();
-        if (result != null) {
-            result += LIVE_PICTURE;
-        }
-        return result;
-    }
-
-    /**
-     * Get the url for the live stream depending wether local or not
-     *
-     * @return Url of the live stream
-     */
-    private String getLiveStreamURL() {
-        String result = getVpnUrl();
-        if (result == null) {
-            return null;
-        }
-
-        StringBuilder resultStringBuilder = new StringBuilder(result);
-        resultStringBuilder.append("/live/index");
-        if (isLocal()) {
-            resultStringBuilder.append("_local");
-        }
-        resultStringBuilder.append(".m3u8");
-        return resultStringBuilder.toString();
-    }
-
-    @SuppressWarnings("null")
-    private String getVpnUrl() {
-        return (module == null) ? null : module.getVpnUrl();
-    }
-
-    public String getStreamURL(String videoId) {
-        String result = getVpnUrl();
-        if (result == null) {
-            return null;
-        }
-
-        StringBuilder resultStringBuilder = new StringBuilder(result);
-        resultStringBuilder.append("/vod/");
-        resultStringBuilder.append(videoId);
-        resultStringBuilder.append("/index");
-        if (isLocal()) {
-            resultStringBuilder.append("_local");
-        }
-        resultStringBuilder.append(".m3u8");
-        return resultStringBuilder.toString();
-    }
-
-    @SuppressWarnings("null")
-    private boolean isLocal() {
-        return (module == null || module.getIsLocal() == null) ? false : module.getIsLocal();
-    }
-}
+/**

```

Comment:
looks like all the line endings have been changed

Reply:
I have now created a new pull-request with squashed commits, that solved also the annoying line-ending problems. See #7927